### PR TITLE
Milestone: Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by kemar-art**
Checking to see if I have an understanding of how GitHub works.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Linters
```
Comment (leolpaz) [inline .github/workflows/linters.yml:1]: - [ ] Kindly make sure that all Linter checks are passing
- [ ] Please use the correct Linter file, you seem to be using the file for HTML, CSS, and JS while this is an HTML and CSS project. I'll leave you with the [link](https://github.com/microverseinc/linters-config/tree/master/html-css) for the repo with the correct Linter

Note: For the first project, setting up the Linter and not having Linter errors are passed as a requirement, but take care as in future projects those would invalidate your review request.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (leolpaz) [inline .gitignore:1]: - [ ] Your .gitignore file should contain entries to ignore both the test.md file and the node_modules folder, the node_modules folder should always be there to not get pushed by accident when setting up a local Linter. Kindly add those to the .gitignore, here is a [link](https://www.pluralsight.com/guides/how-to-use-gitignore-file) on how the .gitignore works.

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @kemar-art , 

You've done an amazing job so far, I'd like to highlight the following:
- Pull request title is descriptive
- README template was used

Some issues need to be resolved before your approval, but you're almost there!


## Required Changes ♻️

- [ ] The .html file, .css file should be on this branch for approval by the reviewer. Kindly add them.
- [ ] The branch should have a descriptive name, as it's required in the GitHub Flow rules linked in the project requirements, it'd be better to change the name of the branch from future to something more descriptive:
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kemar-art** [comment]
@leolpaz

I have made the changes but I am not seeing how to resubmit the link.

Sent from my iPhone

On 23 Nov 2021, at 1:34 PM, Leonardo P. ***@***.***> wrote:

﻿

@leolpaz requested changes on this pull request.

Hi @kemar-art<https://github.com/kemar-art> ,

You've done an amazing job so far, I'd like to highlight the following:

  *   Pull request title is descriptive
  *   README template was used

Some issues need to be resolved before your approval, but you're almost there!

Required Changes ♻️

  *    The .html file, .css file should be on this branch for approval by the reviewer. Kindly add them.
  *    The branch should have a descriptive name, as it's required in the GitHub Flow rules linked in the project requirements, it'd be better to change the name of the branch from future to something more descriptive:
[image]<https://user-images.githubusercontent.com/73140461/143082833-e88bbd8f-789f-4f76-8728-661f4492e78d.png>

[https://camo.githubusercontent.com/ea601010ed4ac26655ecf8ef2441aea1c7d8b3ab5cc23c495c981ea928fb2043/68747470733a2f2f6d65646961342e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966]<https://camo.githubusercontent.com/ea601010ed4ac26655ecf8ef2441aea1c7d8b3ab5cc23c495c981ea928fb2043/68747470733a2f2f6d65646961342e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

________________________________

As described in the Code reviews limits policy<https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form<https://airtable.com/shrQAqnBwek5a0O0s>.

________________________________

In .github/workflows/linters.yml<https://github.com/kemar-art/Hello-Microverse/pull/1#discussion_r755396630>:

> @@ -0,0 +1,62 @@
+name: Linters


  *    Kindly make sure that all Linter checks are passing
  *    Please use the correct Linter file, you seem to be using the file for HTML, CSS, and JS while this is an HTML and CSS project. I'll leave you with the link<https://github.com/microverseinc/linters-config/tree/master/html-css> for the repo with the correct Linter

Note: For the first project, setting up the Linter and not having Linter errors are passed as a requirement, but take care as in future projects those would invalidate your review request.

________________________________

In .gitignore<https://github.com/kemar-art/Hello-Microverse/pull/1#discussion_r755399791>:

> @@ -0,0 +1 @@
+


  *    Your .gitignore file should contain entries to ignore both the test.md file and the node_modules folder, the node_modules folder should always be there to not get pushed by accident when setting up a local Linter. Kindly add those to the .gitignore, here is a link<https://www.pluralsight.com/guides/how-to-use-gitignore-file> on how the .gitignore works.

**Pull request closed without merging**